src/__tests__/providerCollection.spec.ts

```diff
@@ -470,3 +471,84 @@ describe("GET /:provider/collections/:collectionId", () => {
     });
   });
 });
+
+describe("GET /ALL/collections", () => {
+  describe("given the ALL catalog", () => {
+    it("returns collections from multiple providers", async () => {
```

Comment:
This test says multiple providers, however it is only showing one provider PROV1, can we add at least a 2nd provider?